### DISCUSSION TITLE
Thermal Hydraulics Module Exodus File Error

### DISCUSSION THREAD

**Discussion by BenjaminWillisINL** [idaholab/moose#21473] [Q&A Modules: General] [answered]
Hello, I have been trying to access my exodus file output from my most recent input file utilizing the thermal hydraulics module in MOOSE. I have not been able to access the exodus file in either Paraview or Peacock. Every time I go to open the file Paraview or Peacock both crash and I am hit with the error message "paraview quit unexpectedly." I am also getting this error when I try to run the thermal hydraulics module example files. My input file is set up correctly as the I am getting a "solve converged" message in my terminal. Is there any solution where I am able to view my exodus files from the thermal hydraulics module without Paraview crashing on me? Thank you.

**Comment by GiudGiud** [comment, marked as answer]
Hello

What version of paraview is this?

Guillaume

**Reply by BenjaminWillisINL** [reply]
I am using Paraview 5.10.1

**Reply by BenjaminWillisINL** [reply]
More specifically, the version of paraview I downloaded was:

ParaView-5.10.1-MPI-OSX10.13-Python3.9-x86_64.dmg

I am also using a MacBook Pro. Thanks!

**Reply by GiudGiud** [reply]
We are currently having issues with this version.
Please install 5.9 or use the legacy exodus reader in 5.10.1

**Reply by BenjaminWillisINL** [reply]
That worked. Thank you for your help!
